### ISSUE TITLE
Remote image loading issue

### ISSUE THREAD

**Issue by cpdurham** [facebookresearch/visdom#80]
I have an visdom environment with large images in environment '0' and I'm accessing the server remotely. When I load the environment '0', the first image takes a long time to load and loads in '0'. If I then change to environment '1' on the webpage, the rest of the images load in the environment '1', even though they were assigned to '0'. No big deal, just wanted to file a report for it.

**Comment by ajabri** [comment]
Hi @cpdurham !

Yes you're right. I've observed this myself as well. I think the socket should be flushed when the env is changed. Another option is to just refresh the page. I'll get to this sometime soon -- unless you want to give it a try? ;)

**Comment by JackUrb** [comment]
To anyone who wants to try to fix this, a really simple method is to just ignore an incoming window when it arrives at the socket if it doesn't belong to the current env.

https://github.com/facebookresearch/visdom/blob/530e786b9a8005bfbac24092cd52ea6d0f4e3a81/js/main.js#L262

**Comment by Aryanamish** [comment]
I am trying to solve this bug but I don't think you can do it in the main.js because the server is not sending the envID with the window object so will have to change the backend as well, to fix this issue

Data sent from he backend through websocket
```{"command":"window","id":"window_3b5444d7cdd4a4","title":"","inflate":true,"width":null,"height":null,"contentID":"ffc60a8d-a167-4565-bac3-104fdeef1909","content":"This is enviroment 0","type":"text","i":0}```

**Comment by vedansh-5** [comment]
@tonypzy is this issue still valid or was it solved in PR #1267

**Comment by tonypzy** [comment]
> [@tonypzy](https://github.com/tonypzy) is this issue still valid or was it solved in PR [#1267](https://github.com/fossasia/visdom/pull/1267)

Yeah. It has been resolved. The point is rendering across environemnts.
